pyvista/plotting/renderer.py

```diff
@@ -1136,11 +1136,30 @@ def enable_parallel_projection(self):
         often useful when viewing images or 2D datasets.
 
         """
+        # Fix the 'reset camera' effect produced by the VTK when parallel
+        # projection is enabled.
+        angle = np.radians(self.camera.view_angle)
+        self.camera.parallel_scale = self.camera.distance * np.sin(.5 * angle)
+
         self.camera.enable_parallel_projection()
         self.Modified()
 
     def disable_parallel_projection(self):
         """Reset the camera to use perspective projection."""
+        # Fix the 'reset camera' effect produced by the VTK when parallel
+        # projection is disabled.
+        # FIXME: `self.camera.GetFocalPoint()` and `self.camera.focal_point`
+        # are producing different results.
+        focus = self.camera.GetFocalPoint()
```

Comment:
**Observation:** I used `self.camera.GetFocalPoint()` instead of `self.camera.focal_point` because `self.camera.focal_point` is producing a different result than `self.camera.GetFocalPoint()`.

Reply:
It's because we only grab the focal point once.  Lets fix this in this PR and then you can check it.